src/frontends/onnx/tests/onnx_import.in.cpp

```diff
@@ -6303,3 +6303,29 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_model_mish_activation) {
 
     test_case.run_with_tolerance_as_fp(0.000001f);
 }
+
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_multinomial_13_param_inputs) {
+    auto model = convert_model("multinomial.onnx");
+
+    auto test_case = ov::test::TestCase(model, s_device);
+
+    test_case.add_input<float>({0.1f, 0.2f, 0.3f, 0.4f});
+    test_case.add_input<int>({3});
+
+    test_case.add_expected_output<int>({1, 0, 2, 1});
```

Comment:
As I understand, we are expecting [batch, num_samples], in case we have input shape {1,4} and samples = 1 we are expecting {1,1}... Another question - how to verify output, I'm not sure it is correct way.